docs/pages/dsHDMIIn/ds-hdmi-in_L3_Test-Procedure.md

```diff
@@ -0,0 +1,899 @@
+  - [dsHdmiIn_test5_AllmChangeCallback_Verify.py](#dshdmiin_test5_allmchangecallback_verifypy)
+  - [dsHdmiIn_test6_AVlatncyCallback_Verify.py](#dshdmiin_test6_avlatncycallback_verifypy)
+  - [dsHdmiIn_test7_AVIChangeCallback_Verify.py](#dshdmiin_test7_avichangecallback_verifypy)
+  - [dsHdmiIn_test8_SelectPortAndVerifyPortStatus.py](#dshdmiin_test8_selectportandverifyportstatuspy)
+  - [dsHdmiIn_test9_ScaleVideoAndVerify.py](#dshdmiin_test9_scalevideoandverifypy)
+  - [dsHdmiIn_test10_ZoomModeAndVerify.py](#dshdmiin_test10_zoommodeandverifypy)
+  - [dsHdmiIn_test11_GetEDIDInfo_Verify.py](#dshdmiin_test11_getedidinfo_verifypy)
+  - [dsHdmiIn_test12_GetSpdInfo_Verify.py](#dshdmiin_test12_getspdinfo_verifypy)
+  - [dsHdmiIn_test13_SetAndGetEDIDVersion.py](#dshdmiin_test13_setandgetedidversionpy)
+  - [dsHdmiIn_test14_SetAndGetEDID2ALLMSupport.py](#dshdmiin_test14_setandgetedid2allmsupportpy)
+
+## Acronyms, Terms and Abbreviations
+
+- `HAL`    - Hardware Abstraction Layer
+- `L3`     - Level 3 Testing
+- `OEM`    - Original Equipment Manufacture
+- `SoC`    - System on a Chip
+- `HDMI`   - High-Definition Multimedia Interface
+- `SDR`    - Standard Dynamic Range
+- `EDID`   - Extended Display Identification Data
+- `ALLM`   - Auto Low Latency Mode
+- `AVI`    - Audio Video Interleave
+- `SPD`    - Source Product Descriptor
+
+## Setting Up Test Environment
+
+To execute `HAL` `L3` Python test cases, need a Python environment. Follow these steps mentioned in [HPK Public Documentation](https://github.com/rdkcentral/rdk-hpk-documentation/blob/main/README.md)
+
+### Update Configuration Files
+
+#### Rack Configuration File
+
+Example Rack configuration File: `ut/host/tests/configs/example_rack_config.yml`
+
+For more details refer [RAFT](https://github.com/rdkcentral/python_raft/blob/1.0.0/README.md) and [example_rack_config.yml](https://github.com/rdkcentral/python_raft/blob/1.0.0/examples/configs/example_rack_config.yml)
+
+In this file, update the configuration to define the console sessions for the `DUT` and the outbound settings:
+
+|Console Session|Description|
+|---------------|-----------|
+|default|Downloads the streams required for test cases|
+|ssh_player|Plays the stream required for test case|
+|ssh_player_secondary|Plays a secondary stream, if required for test case|
+|ssh_hal_test|Executes the `HAL` binary for the test case|
+
+```yaml
+rackConfig:
+  - dut:
+      ip: "XXX.XXX.XXX.XXX"  # IP Address of the device
+      description: "tv device under test"
+      platform: "tv"
+      consoles:
+        - default:
+            type: "ssh"
+            port: 10022
+            username: "root"
+            ip: "XXX.XXX.XXX" # IP address of the device
+            password: ' '
+        - ssh_player:
+            type: "ssh"
+            port: 10022
+            username: "root"
+            ip: "XXX.XXX.XXX" # IP address of the device
+            password: ' '
+        - ssh_player_secondary:
```

Comment:
Is really used in HDMI in? if not used please remove it

Reply:
done